kind/main.tf

```diff
@@ -13,5 +13,9 @@ module "loki-stack" {
   ingress          = var.ingress
   enable_filebeat  = var.enable_filebeat
 
+  sensitive_values = merge({
+    "loki-distributed.loki.storageConfig.aws.secret_access_key" = var.logs_storage.secret_access_key
```

Comment:
Same problem as the other comment. There is no complete secret for the legacy Loki.